packages/unplugin/src/index.ts

```diff
@@ -94,12 +96,8 @@ const unplugin = createUnplugin<Options>((originalOptions, unpluginMetaContext)
     options.org
   );
 
-  if (options.telemetry) {
-    // eslint-disable-next-line no-console
-    console.log("[Sentry-plugin]", "Sending error and performance telemetry data to Sentry.");
-    // eslint-disable-next-line no-console
-    console.log("[Sentry-Plugin]", "To disable telemetry, set `options.telemetry` to `false`.");
-  }
```

Comment:
I think we shouldn't get rid of this if condition here. We only want to log this, if Sentry is enabled

Reply:
Indeed. However, when creating sentry client inside the logger, we are passing `!!options.telemetry` which dictates whether something will be logged out or not. But you gave me a good idea for a test case!